pkg/cgroups/cgroups.go

```diff
@@ -180,6 +180,14 @@ func (subsys Subsystems) AddPID(name string, pid int) error {
 	return appendIntToFile(filepath.Join(subsys.CPU, name, "tasks"), pid)
```

Comment:
github won't let me comment in the hidden part of the diff, but I would remove or replace `AddPID` as part of this change. Right now it still uses `tasks`.